news/2024-08-10-sunsetting-pypy.md

```diff
@@ -0,0 +1,44 @@
+# Sunsetting PyPy support
+
+TL;DR: We are planning to remove all PyPy builds in conda-forge in a few weeks,
```

Comment:
The original makes it sound like packages will be removed, I think the intention is to stop building new PyPy packages, correct?
```suggestion
TL;DR: We are planning to stop building new releases of packages for PyPy and
remove PyPy from conda-forge feedstock recipes in a few weeks,
```

Reply:
This is a good clarification, thanks. I'm just too packaging-brained, where "builds" are obviously new (because we don't ever remove artefacts).